File: src/main/java/chat/teco/tecochat/like/chatlike/domain/ChatLikeRepository.java
package chat.teco.tecochat.like.chatlike.domain;

import java.util.Optional;
import org.springframework.data.jpa.repository.JpaRepository;

public interface ChatLikeRepository extends JpaRepository<ChatLike, Long> {

    Optional<ChatLike> findByMemberIdAndChatId(Long memberId, Long chatId);
}

File: src/main/kotlin/chat/teco/tecochat/domain/chatlike/ChatLikeRepository.kt
package chat.teco.tecochat.domain.chatlike

import org.springframework.data.jpa.repository.JpaRepository

interface ChatLikeRepository : JpaRepository<ChatLike, Long> {

    fun findByMemberIdAndChatId(memberId: Long, chatId: Long): ChatLike?
}
